### ISSUE TITLE
Support Ctrl+N, Ctrl+T, and Ctrl+W keybindings in chrome / firefox

### ISSUE THREAD

**Issue by vinokurig** [eclipse-theia/theia#6755]
### Description
 
Find a way to override `Ctrl+N`, `Ctrl+T`, and `Ctrl+W` keybinding in chrome / firefox browsers.
### Reproduction Steps

1. Apply [VsCode Emacs Keymap extension](https://marketplace.visualstudio.com/items?itemName=lfs.vscode-emacs-friendly)
2. open some file and try to execute the keybindings.

**WRONG:** Binded browser actions are executed instead of registered actions.
**OS and Theia version:**

**Diagnostics:**

**Comment by azatsarynnyy** [comment]
AFAIK, there's no way to override the reserved hot-keys in Chrome.
See this veeeery old issue https://bugs.chromium.org/p/chromium/issues/detail?id=119881#c84

**Comment by akosyakov** [comment]
@azatsarynnyy that's true, i was thinking about having alternative keybindings for browsers. VS Online should have the same issue. It would be good to start a discussion on VS Code repo to have it in package.json keybindings. Maybe as well on vim extension repo, simply explain that we are trying to run it in Theia, that keybindings don't work in browser and suggest to add alternative  attribute to keybindings.

**Comment by iamalexchip** [comment]
Using `@theia/keymaps` I was able to overrride

- `ctrl + w`
- `ctrl + n`
- `ctrl + pageup`

**Comment by iamalexchip** [comment]
Using `@theia/keymaps` I was able to override the following when using chrome in fullscreen mode.
- `ctrl + w`
- `ctrl + n`
- `ctrl + o`
- `ctrl + pageup`
- `ctrl + pagedown`

**Comment by danepowell** [comment]
Has anyone made progress on this?

I use terminal applications like nano and emacs that use key chords like ctrl + x / ctrl + w. These get intercepted by Chrome, which makes it impossible to use them in Theia. If the problem was just the Theia editor, I could rebind the keys in Theia, but the problem is specifically the terminal. I can't think of any workaround 😢
